akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -333,65 +343,15 @@ final class ActorMaterializerSettings private (
   val maxFixedBufferSize:          Int,
   val syncProcessingLimit:         Int,
   val ioSettings:                  IOSettings,
-  val streamRefSettings:           StreamRefSettings) {
+  val streamRefSettings:           StreamRefSettings,
+  val blockingIoDispatcher:        String) {
 
   require(initialInputBufferSize > 0, "initialInputBufferSize must be > 0")
   require(syncProcessingLimit > 0, "syncProcessingLimit must be > 0")
 
   requirePowerOfTwo(maxInputBufferSize, "maxInputBufferSize")
   require(initialInputBufferSize <= maxInputBufferSize, s"initialInputBufferSize($initialInputBufferSize) must be <= maxInputBufferSize($maxInputBufferSize)")
 
-  // backwards compatibility when added IOSettings, shouldn't be needed since private, but added to satisfy mima
-  def this(
```

Comment:
um, hopefully nobody is using these accidentally from Java

Reply:
I was under the misconception that `class A private()` would give a private constructor also for Java, but verifying with javap I see now that I was wrong. I'll bring them back :/

Reply:
Also, the auxiliary constructors were public, so another reason to bring them back.